nixos/modules/services/desktops/geoclue2.nix

```diff
@@ -125,7 +127,7 @@ in
 
       geoProviderUrl = mkOption {
         type = types.str;
-        default = "https://location.services.mozilla.com/v1/geolocate?key=geoclue";
+        default = mozillaLocationServices;
```

Comment:
```suggestion
```

I would suggest changing the defaults given that MLS is defunct and no good alternative exists yet.

So change the default of `enableWifi` and `enable3G` to `false`. Users won't mind because their location service is already broken.

And remove the default URL, but do this conditionally based on `stateVersion` because otherwise it would potentially introduce errors to existing configurations.

```nix
  # ... snip ...
  ##### implementation
  config = mkIf cfg.enable {

    services.geoclue2.geoProviderUrl = lib.mkIf (lib.versionOlder config.system.stateVersion "24.11") (lib.mkOptionDefault mozillaLocationServices);

    # ... snip ...
  }
```